server/src/main/java/org/eclipse/openvsx/migration/ExtractResourcesJobService.java

```diff
@@ -89,13 +90,13 @@ public Path getExtensionFile(Map.Entry<FileResource, byte[]> entry) {
             var download = entry.getKey();
             var storage = getStorage(download);
             var uri = storage.getLocation(download);
-            restTemplate.execute(uri, HttpMethod.GET, null, response -> {
+            backgroundRestTemplate.execute("{extractResourceUri}", HttpMethod.GET, null, response -> {
```

Comment:
because it will cause high cardinality in metrics leading to very high cost of storing them
each resource uri will be tracked like a route, in such way there is only one route (`extractResourceUri`)